flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/utils/AggregateUtil.scala

```diff
@@ -505,7 +503,7 @@ object AggregateUtil extends Enumeration {
       case VARCHAR | CHAR => fromLegacyInfoToDataType(BinaryStringTypeInfo.INSTANCE)
```

Comment:
Can you change this to `DataTypes.VARCHAR(precision).bridgeTo(BinaryString.class)`, I think that is the right way.

Reply:
+1 
I think this is legacy code.